CHANGELOG.md

```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
 
 <!-- New entries go after this line -->
 
+## v2.7.1 - 2018-02-18
+
+For more details, [see the full changelog](https://github.com/thelounge/lounge/compare/v2.7.0...v2.7.1) and [milestone](https://github.com/thelounge/lounge/milestone/29?closed=1).
+
+This releases mainly fixes bugs that were introduced by previous versions, and comes with very minor improvements to the UI.
+This simply ensures we did not leave any unattended bugs before going for The Lounge v3.
```

Comment:
Mention that we fixed unread markers bug which in turn also fixes memory leak if you had lounge open for long periods of time (e.g. overnight)